tests/integration/test_fast_color_refine.py

```diff
@@ -1,33 +1,32 @@
 """
-Test file for Color Refinement Algorithm
```

Comment:
Heb je nu een nieuwe tests gemaakt over de oude overschreven?
Want dat laatste is niet handig.

Reply:
Nee, als het goed is heb ik een nieuwe file gemaakt die er nog niet was. Heb wel gewoon de code gecopypaste en de naam van het bestand is ook anders dus dat zou niet zo moeten zijn

Reply:
Ah toppie, dan zeg het er in de git diff gewoon raar uit.